README.md

```diff
@@ -140,8 +139,6 @@ can be useful for setups with multiple configs that share common options for
       filename: './output.js',
       path: path.resolve(__dirname),
     },
-    plugins: [new NamedModulesPlugin()],
-    serve: {},
```

Comment:
Was this removal intentional ? :)

Reply:
Yes, to reduce the example, but now that you mention it, it's good to have it even if it's empty.